docker-for-mac/multi-arch.md

```diff
@@ -54,3 +54,134 @@ Notice that this time, the `uname -a` output shows `armv7l` and
 
 Multi-architecture support makes it easy to build <a href="https://blog.docker.com/2017/11/multi-arch-all-the-things/" target="_blank">multi-architecture Docker images</a> or experiment with ARM images and binaries from your Mac.
 
+## Buildx (Experimental)
+
+With the Docker Desktop 2.0.4.0 Edge release, Docker is making it easier than ever to develop containers on, and for Arm servers and devices. Using the standard Docker tooling and processes, you can start to build, push, pull, and run images seamlessly on different compute architectures. Note that you don't have to make any changes to Dockerfiles or source code to start building for Arm.
+
+Docker Desktop 2.0.4.0 Edge release introduces a new CLI command called `buildx`.  Buildx allows you to locally build multi-arch images, link them together with a manifest file, and push them all to a registry using a single command.  With the included emulation, you can transparently build more than just native images.  Buildx accomplishes this by adding new builder instances based on BuildKit, and leveraging Docker Desktop's technology stack to run non-native binaries.
+
+### Install
+
+1. Download Docker Desktop 2.0.4.0 or higher.
+
+    - [Docker Desktop 2.0.4.0 for Mac](https://docs.docker.com/docker-for-mac/edge-release-notes/)
+    - [Docker Desktop 2.0.4.0 for Windows](https://docs.docker.com/docker-for-windows/edge-release-notes/)
+
+1. Follow the on-screen instructions to complete the installation process. After you have successfully installed Docker Desktop, you will see the Docker icon in your task tray.
+
+1. Click **About Docker Desktop** from the Docker menu and ensure you have installed Docker Desktop version 2.0.4.0 (33772) or higher.
+
+![about-docker-desktop-buildx](./images/desktop-buildx-version.png)
+
+### Build and run multi-architecture images
+
+Run the command `docker buildx ls` to list the existing builders. This displays the default builder, which is our old builder.
+
+```bash
+$ docker buildx ls
+
+NAME/NODE DRIVER/ENDPOINT STATUS  PLATFORMS
+default * docker
+  default default         running linux/amd64, linux/arm64, linux/arm/v7, linux/arm/v6
+```
+
+Create a new builder which gives access to the new multi-architecture features.
+
+```bash
+$ docker buildx create --name mybuilder
+
+mybuilder
+```
+
+Switch to the new builder and inspect it.
+
+```bash
+$ docker buildx use mybuilder
```

Comment:
You can add a note for `docker buildx create --name mybuilder --use` allows to create a builder and switch to it in one command.

Reply:
Added a statement about creating a builder and switching it using a single command.